### PR TITLE
[Format] add period suffix to comment field and fix issue

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
fixes: [now support script yml](https://jira-hq.paloaltonetworks.local/browse/CIAC-8308)
fixed: [added a new line with a period](https://jira-hq.paloaltonetworks.local/browse/CIAC-8281)

## Description
fixed an issue where the format  added a new line with a period
when the description start with pipe `|`
```
comment: |-
  The test
 ```

### CODE REVIEW

**Review by JasBeilin — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 * Fixed an issue where **validate** falsly failed with error `DS108` on descriptions ending with a newline.
+* Added formatting for script yml files when period is missing in the end of comment field, in the **format** command.
+* Fixed an issue where **format**  add a newline with a period when the description field missing a period.
 * The content graph will now include the **python_version** field that each script/integration uses.
 * Updated the **update-release-notes** command message structure when is run with **--force** flag.
 * Added support for lazy loading the of widgets in XSIAM dashboards.
 
 ## 1.20.1
-* Added formatting for yml files when period is missing in the end of description field, in the **format** command.
+* Added formatting for integration yml files when period is missing in the end of description field, in the **format** command.
```
Comment (JasBeilin) [inline CHANGELOG.md:12]: Are we ok with changing it after release?

File: demisto_sdk/commands/format/tests/test_formatting_generic_test.py [hunk 2]
```diff
@@ -274,48 +274,97 @@ def test_initiate_file_validator(mocker, is_old_file, function_validate):
...
+            "description without dot and empty string in the end ",
+            "description without dot and empty string in the end.",
+        ),
+        (
+            "comment with dot and 'new_line' in the end. \n",
+            "comment with dot and 'new_line' in the end.",
+            "description with dot and 'new_line' in the end. \n",
+            "description with dot and 'new_line' in the end.",
+        ),
+        (
```
Comment (kobymeir) [inline demisto_sdk/commands/format/tests/test_formatting_generic_test.py:328]: comment why the \n

**Review by kobymeir — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (shmuel44) [reply CHANGELOG.md:12]: you are right, i will del this

File: demisto_sdk/commands/format/tests/test_formatting_generic_test.py (hunk 2 above)
Comment (shmuel44) [reply demisto_sdk/commands/format/tests/test_formatting_generic_test.py:328]: Done

**Pull request merged**